mlocal/frags/build_cli.mk

```diff
@@ -31,8 +31,13 @@ $(apptainer_INSTALL): $(apptainer)
 	$(V)umask 0022 && mkdir -p $(@D)
 	$(V)install -m 0755 $(apptainer) $(apptainer_INSTALL) # set cp to install
 
+singularity_INSTALL := $(DESTDIR)$(BINDIR)/singularity
+$(singularity_INSTALL):
```

Comment:
Can we have an `mconfig` option to opt-in/out of the link?

Reply:
Upon further thought, the overwriting of the `run-singularity` script, which is required for the expected behavior of directly executable SIF images, is enough to make the installation old singularity versions and apptainer at the same prefix problematic. (was my initial idea with this mconfig option)